inst/private/python_ipc_pytave.m

```diff
@@ -70,7 +70,6 @@
 
   s = strjoin(cmd, newl);
   pyexec(s)
-  
-  info.raw = pyeval('str(_outs)');
```

Comment:
On 08/06/16 22:28, Abhinav Tripathi wrote:
> If we are removing info. We should remove |info.prelines| too (you
> probably missed it)

No I think its needed for error messages.  The error messages might be
wrong, but for now we need it.